### PR TITLE
Added certificate generation feature with upload and download functionality; updated navigation and improved team image upload instructions.

### PR DESCRIPTION
**Pull request by simboeusu**

### CODE REVIEW

File: src/controllers/CertificateController.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    doc.render(placeholders);
+
+    const buffer = doc.getZip().generate({ type: 'nodebuffer' });
+
+    // Save the certificate to the server
+    //Check if the folder exists, if not create it
+    if (!fs.existsSync(path.join(__dirname, '../../public/certificates/'))) {
+        fs.mkdirSync(path.join(__dirname, '../../public/certificates/'), { recursive: true });
+    }
+    const outputPath = path.join(__dirname, '../../public/certificates/', `${team.name}_certificate.docx`);
```
Comment (simboeusu) [inline src/controllers/CertificateController.js]: Evt. noch Platzierung in Dateinamen einfügen

File: src/views/layouts/certificate.hbs [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+                {{#each teams}}
+                    <option value="{{this._id}}">{{this.name}} (Platz: {{this.rank}})</option>
+                {{/each}}
+            </select>
+        </div>
+        <button type="submit" class="btn btn-primary mt-3">Urkunde generieren</button>
+    </form>
+</div>
+<script>
+document.getElementById('teamSearch').addEventListener('input', function() {
```
Comment (mo374z) [inline src/views/layouts/certificate.hbs:28]: Does not work entirely the first shown value of the dropdown is not refreshed when we search for a team

Maybe do it similarly to searching players to assign goals (list of matches above, which then gets filtered)

File: src/views/layouts/main.hbs [hunk 3]
```diff
@@ -66,6 +64,9 @@
                             <li class="nav-item">
                                 <a class="nav-link ms-5" href="/player">Spieler</a>
                             </li>
+                            <li class="nav-item">
+                                <a class="nav-link ms-5" href="/certificate">Urkunden</a>
```
Comment (mo374z) [inline src/views/layouts/main.hbs:68]: Dont know if we need it here - can be placed in Setting in my opinion keeping the header not to full of items

File: src/views/layouts/certificate.hbs (hunk 2 above)
Comment (simboeusu) [reply src/views/layouts/certificate.hbs:28]: Fixed !

File: src/views/layouts/main.hbs (hunk 3 above)
Comment (simboeusu) [reply src/views/layouts/main.hbs:68]: Ok i have made a Card for Klinger but this is a quick change later where the certificate page point is

**Pull request merged**